hydromt_wflow/workflows/soilparams.py

```diff
@@ -83,7 +83,7 @@ def calc_kv_at_depth(depth, kv_0, f):
 
 
 def calc_kvfrac(kv_depth, target):
```

Comment:
I guess for this function you can keep kvfrac everywhere (it's an internal function anyhow)